pyhive/presto.py

```diff
@@ -79,7 +80,7 @@ class Cursor(common.DBAPICursor):
 
     def __init__(self, host, port='8080', username=None, catalog='hive',
                  schema='default', poll_interval=1, source='pyhive', session_props=None,
-                 protocol='http', password=None):
+                 protocol='http', password=None,auth='NONE',verify=False):
```

Comment:
I put it as False because usually users would use it without SSL but I can set it to True by default if that is preferred.